### PR TITLE
[Dashboard] Remove ListItem (1) and fix some UI issues from the aspect-ratio PR

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on updating various components to replace `List` and `ListItem` from `@chakra-ui/react` with standard HTML `<ul>` and `<li>` elements for better semantic HTML structure. It also includes the removal of the `ListLabel` component in favor of a more detailed inline structure for labels.

### Detailed summary
- Deleted `list-label.tsx` component.
- Replaced `List` and `ListItem` with `<ul>` and `<li>` in `EarnReasonSection`, `ReadyPlayer3Landing`, `Hackathon`, `HackathonEarn`, `AirdropUploadERC20`, and `AirdropUpload` components.
- Updated `CreateListingsForm` to render NFT details directly within a `Card`.
- Adjusted the `ContactUs` component to use `<Flex>` for layout instead of `List`.
- Removed `List` and `ListItem` imports from various components to clean up unused code.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
* **#4846** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4846?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx [hunk 1]
```diff
@@ -50,6 +50,7 @@ import type {
 } from "thirdweb/extensions/marketplace";
 import { useActiveAccount, useSendAndConfirmTransaction } from "thirdweb/react";
 import {
+  Card,
```
Comment (jnsdls) [inline apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx:53]: do not add new imports from `tw-components` - we are removing this, there should be a `card` component in shadcn components we already use

File: apps/dashboard/src/contract-ui/tabs/nfts/components/airdrop-upload.tsx [hunk 2]
```diff
@@ -18,13 +17,12 @@ import {
   Thead,
   Tooltip,
   Tr,
-  UnorderedList,
 } from "@chakra-ui/react";
 import { Logo } from "components/logo";
+import { Upload } from "lucide-react";
```
Comment (jnsdls) [inline apps/dashboard/src/contract-ui/tabs/nfts/components/airdrop-upload.tsx:22]: `UploadIcon`

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/pages/hackathon/earn.tsx [hunk 3]
```diff
@@ -324,24 +318,22 @@ const HackathonEarn = () => {
                   padding="0 22px"
                 >
                   <List color="white" styleType="none" maxW={807} spacing={3}>
```
Comment (MananTank) [inline apps/dashboard/src/pages/hackathon/earn.tsx:320]: <List> not removed

File: apps/dashboard/src/pages/hackathon/earn.tsx [hunk 4]
```diff
@@ -220,21 +216,19 @@ const HackathonEarn = () => {
                 </Text>
                 <Flex fontSize={{ base: "18px", md: "20px" }}>
                   <List color="white" styleType="none" spacing={3}>
```
Comment (MananTank) [inline apps/dashboard/src/pages/hackathon/earn.tsx:218]: <List> not removed

File: apps/dashboard/src/pages/hackathon/consumer-crypto.tsx [hunk 5]
```diff
@@ -261,18 +259,18 @@ const Hackathon = () => {
                 </Text>
                 <Text size="body.xl" color="white">
                   <List>
```
Comment (MananTank) [inline apps/dashboard/src/pages/hackathon/consumer-crypto.tsx:261]: <List> not removed

**Review by MananTank — CHANGES_REQUESTED** [review]
The bullet icon is going outside the list container a little bit - Prefer using `<UnorderedList />` or `<OrderedList />` component ( defined in components/ui/List.tsx ) with `<li />` instead

**Review by jnsdls — COMMENTED** [review]
just waiting for manan's feedback to be addressed

**Review by jnsdls — COMMENTED** [review]
LG from my side - will wait for @MananTank to sign off

**Review by MananTank — APPROVED** [review]

**Pull request merged**